src/core/compact_object.cc

```diff
@@ -991,6 +1007,8 @@ void CompactObj::Free() {
     VLOG(1) << "Freeing JSON object";
     u_.json_obj.json_ptr->~JsonType();
     tl.local_mr->deallocate(u_.json_obj.json_ptr, sizeof(JsonType), kAlignSize);
+  } else if (taglen_ == SBF_TAG) {
+    DeleteMR<SBF>(u_.sbf);
```

Comment:
But, by the way, `DeleteMR` calls `deallocate(ptr, alignof(T))` which is not exactly the same function as was used for allocating

Reply:
Good catch, fixed.